res/values/strings.xml

```diff
@@ -319,17 +319,17 @@
     <string name="login_imap_login">IMAP login-name</string>
     <string name="login_imap_server">IMAP server</string>
     <string name="login_imap_port">IMAP port</string>
-    <string name="login_imap_security">IMAP Security</string>
+    <string name="login_imap_security">IMAP security</string>
     <string name="login_outbox">Outbox</string>
     <string name="login_smtp_login">SMTP login-name</string>
-    <string name="login_smtp_password">SMTP Password</string>
+    <string name="login_smtp_password">SMTP password</string>
     <string name="login_smtp_server">SMTP server</string>
     <string name="login_smtp_port">SMTP port</string>
-    <string name="login_smtp_security">SMTP Security</string>
+    <string name="login_smtp_security">SMTP security</string>
     <string name="login_auth_method">Authorization method</string>
     <string name="login_info_oauth2_title">Continue with simplified setup?</string>
     <string name="login_info_oauth2_text">The entered e-mail-address supports a simplified setup (OAuth2).\n\nIn the next step, please allow Delta Chat to act as your Chat with E-Mail app.\n\nThere are no Delta Chat servers, your data stays on your device!</string>
-    <string name="login_info_gmail_text">For GMail Accounts you need to create an App-Password if you have 2FA enabled. If this setting is not available, you need to enable Less Secure Apps.</string>
+    <string name="login_info_gmail_text">For Gmail Accounts you need to create an App-Password if you have 2-Step Verification enabled. If this setting is not available, you need to enable less secure apps.</string>
```

Comment:
- 2FA is "Two-Factor Authentication", however, maybe even the abbreviation is known enough.
- maybe write "Less secure apps" in quotes to make more clear that it is the name of an option.

Reply:
I wrote as it is in Google's documents and Android interface.

Reply:
I wrote it like Google's documents and Android interface.

Reply:
you're right, google does not speak of 2FA at all. but i would put it in quotes, both "2-Step Verification" and "Less secure apps".

and if maybe bring both terms to the same form, so, first character uppercase, rest lowercase, independingly how google writes that (there is often marketing involved, no need to pick that up :)

Reply:
but maybe it is just fine :)